Zemismart/__init__.py

```diff
@@ -174,9 +250,64 @@ def save_battery(self, battery):
         self.battery = battery
 
     def update(self):
+        self._get_position_responses = set()
         if not self.send_Zemismart_packet(self.get_position_cmd, bytearray([0x01]), 1):
             return False
         elif not self.send_Zemismart_packet(self.get_battery_cmd, bytearray([0x01]), 1):
             return False
         else:
             return True
+
+    def timer_toggle(self, timer_id, enabled):
+        if timer_id >= len(self.timers):
+            raise IndexError('Timer #%d not found' % timer_id)
+
+        ret = self.send_Zemismart_packet(self.update_timer_cmd,
+                                         bytearray([timer_id + 1, 0x00, int(enabled), 0x00, 0x00, 0x00, 0x00]),
+                                         1)
+
+        if ret:
+            self.timers[timer_id - 1].enabled = enabled
+
+        return ret
+
+    def timer_delete(self, timer_id):
+        if timer_id >= len(self.timers):
+            raise IndexError('Timer #%d not found' % timer_id)
+
+        ret = self.send_Zemismart_packet(self.update_timer_cmd,
+                                         bytearray([timer_id + 1, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00]),
+                                         1)
+
+        if ret:
+            del self.timers[timer_id - 1]
+
+        return ret
+
+    def _update_timer_internal(self, timer_id, timer):

```

Comment:
I am not sure I get what this name means. What is "interval", and what does it have to do with adding/updating a timer?

Reply:
It's `interNal`, not `interVal` :) I wasn't been able to come up with a better name for a method which will be used from add() and update().

Reply:
Ahh, yes. it is probably fine then.